packages/markdown-docx/src/ToCiceroMarkVisitor.js

```diff
@@ -446,9 +477,18 @@ class ToCiceroMarkVisitor {
                         }
                     }
                 }
+            } else if (subNode.name === 'w:hyperlink') {
+                this.traverseElements(subNode.elements, TRANSFORMED_NODES.link, subNode.attributes['r:id']);
             } else if (subNode.name === 'w:r') {
                 let nodeInformation = { properties: [], value: '' };
-                this.fetchFormattingProperties(subNode, nodeInformation);
+                if (parent === TRANSFORMED_NODES.link) {
+                    nodeInformation.linkId = id;
+                }
+                this.fetchFormattingProperties(
+                    subNode,
+                    nodeInformation,
+                    parent === TRANSFORMED_NODES.link ? TRANSFORMED_NODES.link : TRANSFORMED_NODES.paragraph
```

Comment:
Are there only two types of parents possible? This looks quite incomprehensible and an attempt to hardcode the logic. If it's really required, maybe you can few lines of comments elucidating this.

Reply:
@algomaster99 
Nope, many parents can be possible, but when we call this we only need to check whether this function is called by Link or not (particularly these lines). So I have passed `link` if true else `paragraph`.

The main reason for not naming it as `calledByLink` is because we use similar things in `clause` also.